### ISSUE TITLE
Local MP3 conversion

### ISSUE THREAD

**Issue by morues** [gantt/downloadyoutube#44]
There is an extension for Firefox that does MP3 conversion locally. The important thing is that it does the conversion in background (no UI involved). It would be great if it is integrated to "Download YouTube Videos as MP4".

https://github.com/inbasic/media-converter/blob/master/connect.md

I have done some preliminary tests and seems like you can do the conversion in a few lines of code. The only concern I have is that it is only for Firefox users.

Anyhow I would love to see the integration.

**Comment by gantt** [comment]
Yes, but it uses ffmpeg. I'd like to find a different way because I can't bundle ffmpeg and I don't want to ask users to download it.
